x-pack/legacy/plugins/code/public/style/_monaco.scss

```diff
@@ -1,22 +1,33 @@
 .codeSearch__highlight {
-	background-color: $euiColorVis5;
-	color: black !important;
-	padding: $euiSizeXS / 2;
-	border-radius: $euiSizeXS / 2;
-	font-weight: bold;
-	font-style: oblique;
-	cursor: pointer;
+  background-color: $euiColorVis5;
+  color: black !important;
+  padding: $euiSizeXS / 2;
+  border-radius: $euiSizeXS / 2;
+  font-weight: bold;
+  font-style: oblique;
+  cursor: pointer;
 }
 
 .codeBlock__line--highlighted {
   background-color: $euiColorLightShade;
 }
 
+.code-monaco-highlight-line {
```

Comment:
These styles were deleted on master, I don't think you want to re-add them here. Same with `.code-mark-line-number`